tests/helper.py

```diff
@@ -1,8 +1,18 @@
 from errbot.backends.test import TestBot
+from errbot.plugin_info import PluginInfo
+from errbot.templating import add_plugin_templates_path
+from pathlib import Path
 
-def plugin_testbot(klass, loglevel, config=None):
+def plugin_testbot(klass, plugin_name, loglevel, config=None):
```

Comment:
can't the plugin name be derived using the `klass`?

Reply:
but sometimes the plugin name can be different from the class name. Eg: git_stats plugin

Reply:
There is a solution, load plugin file, get the name key.

Reply:
but even for that, we'll have to pass the name of the plug file.

Reply:
Can you not find the name of the plug file from the plug name?

Reply:
Yes, for labhub. But it is not required to load templates for  Ban and git_stats plugins.